### ISSUE TITLE
Work out layout of the kitchen

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#690]
We need to do this. I think we're resigned to having to put both sinks in the one area for access to drainage. Over in #686 there has been a few ideas about layout of the kitchen but I thought that now needed breaking into its own issue.

@zarino and I will be spending some time on this on Thursday but if anyone wants to get their thoughts heard then please jump in here. I think we're likely to stick with the workshop sink to the left of the kitchen and the other sink and units to the right and we'll just need to keep the workshop sink area tidy but if anyone wants to argue for other arrangements then please do!

**Comment by zarino** [comment]
Big thanks to @johnmckerrell for spending yesterday afternoon with me at the Tapestry, working this puzzle out. (Also to @DoESsean and Martin for helping us lug a sink unit over there!)

The room isn’t wide enough to have two banks of units opposite each other (like we have at the moment). So the most efficient use of space is to line units up against the _far_ wall, under the window.

(Left to right: tall fridge, work sink unit, food sink unit, worktop unit with kettle/toaster/microwave, small fridge with coffee machine)

<img width="928" alt="screen shot 2018-03-16 at 08 33 37" src="https://user-images.githubusercontent.com/739624/37510940-c4596d7c-28f4-11e8-88c0-ba509dd10ef0.png">

Advantages of this:

* Everything fits – with enough space for a wheelchair user, or for two people to pass each other.
* Sink not immediately visible as you enter the door.
* Nice matching bank of three units as you walk into the space.

Disadvantages:

* Tall fridge is quite prominent. Maybe we could "treat" it somehow (paint?) to make a feature out of it? ;-)
* Need to work out where to put the four "overhead" kitchen units, since they can’t go on the window above the sinks etc.

**Comment by johnmckerrell** [comment]
Yeah, tbh I'm still not 100% on this one, mainly due to the lack of options for the overhead cupboards. One thing to mention is that we were undecided on whether the small fridge should go in the corner or one space to the left (so then the last two units left to right would be: small fridge with kettle, etc. on it, unit with no sink).

**Comment by zarino** [comment]
@johnmckerrell For storage, I’m still wondering about putting the four overhead cupboards into a 2×2 arrangement on the west wall (right wall on plan above).

And I’m also wondering whether we could maybe get some very shallow (20–30cm deep) shelves or open-faced cupboards on the north wall (bottom of the plan above), opposite the units, to store things like mugs, plates, etc? We’d need to check they leave enough walking space, though!

**Comment by johnmckerrell** [comment]
And I'm going to keep throwing this in like an after thought - dishwasher?

**Comment by zarino** [comment]
Dammit.

**Comment by johnmckerrell** [comment]
https://www.wikihow.com/Paint-a-Refrigerator

**Comment by zarino** [comment]
@johnmckerrell If you really _do_ want a dishwasher, there _might_ be space for a slimline model (usually ~45cm wide) in that line of units?

**Comment by johnmckerrell** [comment]
Any progress made with whether we could find space to fit the cupboards on the wall? I think I can see what you mean with narrow cupboards but obviously they'd need to be quite narrow.

I like the idea of a slimline dishwasher, would be good to confirm it can fit.

**Comment by Robotorium** [comment]
If it can't and it isn't too much hassle sacrificing the counter space, a
countertop one could work?

Op ma 19 mrt. 2018 11:59 schreef John McKerrell <notifications@github.com>:

**Comment by johnmckerrell** [comment]
Yeah countertop could work. Actually just occurred to me that a chief function of the small fridge is "holding up the coffee maker", we could switch that role onto the dishwasher instead and consider keeping the small fridge somewhere else (e.g. in the events room)

**Comment by zarino** [comment]
+1 for swapping the small fridge with a dishwasher, and moving the small fridge out into the events room. Definitely makes sense to have cold drinks closer to the action during events, rather than either decanting them into a bucket or forcing someone to pass them through from the kitchen. And since that also frees up space for a full-size dishwasher, then doubleplusgood.

@johnmckerrell I wonder whether we’d want the dishwasher closer to the sinks in that case? Maybe _between_ the two sinks? So, from left to right, it’ll go:

* Big fridge
* Work sink (worktop left, sink right)
* Dishwasher and coffee machine
* Food sink (sink left, worktop right)
* Worktop

**Comment by johnmckerrell** [comment]
More of a hunch than anything but I like the look of this one:

https://ao.com/product/dfn28r22b-beko-standard-dishwasher-black-48542-23.aspx

Cutlery trays seem a pretty good option, A++ efficient, black, apparently works out how long to wash by how dirty things are.

**Comment by jackie1050** [comment]
Agree with all of that

On 19 Mar 2018 12:20, Zarino Zappia <notifications@github.com> wrote:

+1 for swapping the small fridge with a dishwasher, and moving the small fridge out into the events room. Definitely makes sense to have cold drinks closer to the action during events, rather than either decanting them into a bucket or forcing someone to pass them through from the kitchen. And since that also frees up space for a full-size dishwasher, then doubleplusgood.

@johnmckerrell<https://github.com/johnmckerrell> I wonder whether we’d want the dishwasher closer to the sinks in that case? Maybe between the two sinks? So, from left to right, it’ll go:

  *   Big fridge
  *   Work sink (worktop left, sink right)
  *   Dishwasher and coffee machine
  *   Food sink (sink left, worktop right)
  *   Worktop

**Comment by zarino** [comment]
> Cutlery trays seem a pretty good option, A++ efficient, black, apparently works out how long to wash by how dirty things are.

The very idea that one could get a dishwasher in black is a revelation to me! Silver would look ok in our case too. Just a shame it won’t match the big fridge. Time to break out the black spray paint ;-)

**Comment by Robotorium** [comment]
I might have a can of chalkboard paint lying around... Fridge with
note-taking facility?

Op ma 19 mrt. 2018 12:57 schreef Zarino Zappia <notifications@github.com>:

**Comment by jackie1050** [comment]
There was a conversation yesterday about chalk painting the partition door to the event space ...

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by johnmckerrell** [comment]
We previously (#469) decided against chalk!

**Comment by Robotorium** [comment]
Oops, sorry! Didn't mean to reignite an old debate.

Op ma 19 mrt. 2018 15:15 schreef John McKerrell <notifications@github.com>:

**Comment by magman2112** [comment]
I would suggest rotating one of the sinks 180 degrees, so that anything potentially spilling out of the work sink doesn’t contaminate the food sink.

I was also thinking that if we had a worktop that extended a little through the hatch into the event space, along the wall by the lift, this would allow anything prepared in the kitchen to be mad easily available in the event space, as well as making a better base for any eventual hatch.

**Comment by johnmckerrell** [comment]
I took some photos while I was in today to get an idea of how the cupboards would be. I did think we might be able to get away with cupboards opposite the units if they had sliding doors.


![2018-03-19 13 29 56](https://user-images.githubusercontent.com/27243/37624300-97f515ec-2bbf-11e8-8aae-49fe31da2e46.jpg)
![2018-03-19 13 28 06](https://user-images.githubusercontent.com/27243/37624301-980b7a08-2bbf-11e8-84f7-b7929c4f4d73.jpg)
![2018-03-19 13 28 00](https://user-images.githubusercontent.com/27243/37624302-9824f820-2bbf-11e8-9675-66a38616e8cb.jpg)

**Comment by DefProc** [comment]
I consulted a interior design expert* and they suggested: "don't put up overhanging cupboards with nothing underneath, unless you're trying to get people to crack their heads open"... 🤕

*not an expert.

They might work better on the 2 short walls.

**Comment by johnmckerrell** [comment]
I had assumed anything opposite the units would be floor-standing, I thought that's what @zarino had said but I see now he just said "open faced". Even on the small walls we still likely end up with nothing below but hopefully there it's less likely you'd hit your head.

@magman2112 we'd need to swap the units and have the sinks at opposite ends as they can't be rotated 180 degrees (or not without moving the tap). Problem potentially then is that the dirty stuff on the worktop could similarly overflow. Perhaps we could fit a small piece of worktop between them to guard against this (or put the dishwasher between them but I'm not sure about that yet.

**Comment by zarino** [comment]
I assumed (after our dishwasher discussions yesterday) that the dishwasher would go between the two sinks. Like this: (DW = dishwasher)

<img width="861" alt="screen shot 2018-03-20 at 07 47 25" src="https://user-images.githubusercontent.com/739624/37641802-f9dab91c-2c12-11e8-916b-cc24cb480fb2.png">

I’ve fancifully included a 40cm deep, built-in "pantry" unit on the left-hand side, just to show how it’d fit, and a set of three 60x20cm shelving / bookcase units on the front wall, again just to see how it’d fit. (I was thinking we could go with something cheap like [IKEA LAIVA (£15)](http://www.ikea.com/gb/en/products/storage-furniture/bookcases/laiva-bookcase-birch-effect-art-60128208/) and just paint them grey or black to match everything else. By my calculation, with units on one side and shelving on the other, you’ll have about 90–100cm of walk space. @Robotorium is that enough room to turn a wheelchair 180 degrees?

@johnmckerrell Remind me – what was our reason for not putting the big fridge at the far right end of the row, in front of the serving hatch? Was it just that, then, the serving hatch would be too tiny?

**Comment by magman2112** [comment]
I suspect we are getting closer to a workable layout here. Next question is where to put power sockets. 

With the window behind most of these cabinets, we are likely to have lower level sockets for most of the cabinets. There is the brick pillar next to the Events space hatch that could host some above counter level sockets though, and possibly some switches for lower level sockets (in a modern house kitchen, washing machines and dishwashers typically have a higher level switch to isolate the socket below). Another option is putting some sockets on the window ledge, slightly below the counter level, behind the counters - the feasibility of this would have to be checked with the electrician though.

Do we also want some above counter sockets near the Event space hatch (for food warmers, water boilers, bread maker, etc.)?

**Comment by zarino** [comment]
@magman2112 ISTR @johnmckerrell’s plan for sockets was 2 at the far left-hand end, near the orange soil pipe, 4 on the brick pillar between the window and the hatch, and 2 on the far right-hand wall (back when we were going to have a fridge over there) – all at the usual, mid-chest height.

That might have to change, though, to accommodate the dishwasher, and whatever we decide to to about a cupboard/pantry on the far left wall. @johnmckerrell?

**Comment by Robotorium** [comment]
It's not enough for mine (106 cm long) but it's plenty for me to go in
comfortably and back out (think it's 66cm wide, not sure but definitely a
fair bit less than 90). It looks to me like the space won't allow for much
more, which is fair enough.

Op di 20 mrt. 2018 08:28 schreef Zarino Zappia <notifications@github.com>:

**Comment by zarino** [comment]
Thanks @Robotorium. I'm not 100% sold on putting stuff along the brick wall yet, so it’s useful to know this when we’re weighing up the pros and cons.

**Comment by johnmckerrell** [comment]
Ok, I've marked up the walls for placing the sockets to match the placement above. I'm not sure we'd want the coffee maker on the dish washer next to the workshop sink but we'll have options.

<img width="861" alt="37641802-f9dab91c-2c12-11e8-916b-cc24cb480fb2" src="https://user-images.githubusercontent.com/27243/37651699-ec07ea4e-2c30-11e8-92b1-4eaff08342ac.png">

**Comment by magman2112** [comment]
Whilst I was in the Tapestry building with @johnmckerrell and @jackie1050 this afternoon I suggested that we may be able to work some standard type worktop material into the area by the event hatch, as mentioned above.

A further enhancement could be to put a lower worktop more suitable for wheelchair users along the lift wall side of the kitchenette. I suspect though that we would have to get all of the cabinets in situ before this could be worked out properly.

It also helps that I have all of the require power tools and jigs for working on typical chipboard kitchen worktops.

**Comment by Thingomy** [comment]
How many network sockets do we need in the kitchen? between the fridge,
cofee machine and any of the other gadgets we may construct, I suspect that
we could need 4, thoughts?

Thanks,
JR

**Comment by DefProc** [comment]
I have a source for a working, free to a good home, slimline dishwasher that's currently located in Nottingham. For collection if you're interested?

**Comment by DefProc** [comment]
Dishwasher delivered

**Comment by amcewen** [comment]
Most of the most recent layout has now been instantiated.  The fridge is waiting until the flooring guys have fitted the edging strip, and there's no pantry or 20cm shelving.  Plus the dishwasher isn't boxed in - see #750 for that.

![](https://user-images.githubusercontent.com/739624/37641802-f9dab91c-2c12-11e8-916b-cc24cb480fb2.png)

We haven't done anything with the old overhead units.  Is there a plan for them?  Or do the pantry and 20cm shelves replace that?  If so, what do we need to do to progress that?

**Comment by zarino** [comment]
I wonder whether we should close this ticket, and instead break out the remaining tasks into separate, more actionable tickets, @johnmckerrell? Perhaps:

* Install worktop above / housing around dishwasher
* Affix the black, overhead kitchen cupboards onto the plasterboard wall to your immedate left as you enter the kitchen (where the bin currently is) – in either a 2x2 configuration at chest-to-head height, or in a tall 1x4 "tower" configuration, depending how much space there is
* Investigate open, narrow profile storage solution for regularly-accessed items, on the brick wall to your immediate right as you enter the kitchen (max 20cm depth, so as not to obstruct the walkway)

?

**Comment by johnmckerrell** [comment]
Sounds reasonable.

**Comment by zarino** [comment]
I’ll do that now then.
